src/css/components/grid.css

```diff
@@ -0,0 +1,15 @@
+.grid-basic {
```

Comment:
@pavelloz has strong opinions about `@apply` and so currently has a community :) pinged offline!

Reply:
Ok, it would probably be a better idea to wrap the whole grid rendering in a separate partial/component instead as we are using the same structure in various places in code. Will be discussed tomorrow on daily.